src/com/cackhainiemcoban/ConditionalLoop/PrintNumberPrimeLessN.java

```diff
@@ -0,0 +1,34 @@
+package com.cackhainiemcoban.ConditionalLoop;
+
+import java.util.Scanner;
+
+public class PrintNumberPrimeLessN {
+    public static boolean conditionalChooseNumberPrime(int n) {
+        if (n ==2 ) {
+            return true;
+        }
+        for (int i = 2; i <= (int) Math.sqrt(n); i++) {
```

Comment:
chỗ này e nên Khai báo 1 biến ví dụ limited = Math.sqrt(n) 
vì khi để trong vòng for có nghĩa em phải tính lại phép toán căn này N lần, mà phép tính căn cũng là 1 phép tính phức tạp. 
Để phép tính này ra ngoài để tăng hiệu suất (performance) của project